src/Internal.AspNetCore.BuildTools.Tasks/Internal.AspNetCore.BuildTools.Tasks.csproj

```diff
@@ -6,19 +6,10 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <DefineConstants>$(DefineConstants);BuildTools</DefineConstants>
     <Description>MSBuild tasks. This package is intended for Microsoft use only</Description>
-
-    <!--
-      The netstandard1.0 TFM doesn't actually compile tasks.
-      It's just there so Internal.AspNetCore.Sdk gets a dependency to this project.
-    -->
-    <TargetFrameworks>netstandard1.6;net46;netstandard1.0</TargetFrameworks>
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Build tasks still need to cross-compile for .NET Framework. You can't use ns2.0 assemblies from MSBuild (full-framework) because the netstandard shims will not be available.

Also, we still need the netstandard1.0 TFM.